public/js/graph/react_graph.js

```diff
@@ -438,6 +440,34 @@ var Graph = React.createClass({
         });
     },
 
+    onButtonPress: function(zoomOrPanFunction, direction) {
+        this.setState({mouseDown: setInterval(() => zoomOrPanFunction(direction), 100)});
```

Comment:
Increase the interval time to 500. 100 is too short, and the `setState` is a bit of a hack right now (interval time doesn't correspond to re-rendering time).